packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.js

```diff
@@ -203,28 +204,45 @@ class MultiFileUploadQuestion extends Question {
 		}
 		const uploadedFiles = await this.#saveFilesToBlobStorage(validFiles, journeyResponse);
 
-		// add saved docs to response
-		const responseToSave = {
+		const journeyFiles = {
 			answers: {
 				[this.fieldName]: {
 					uploadedFiles: []
 				}
 			}
 		};
 
-		responseToSave.answers[this.fieldName].uploadedFiles = [];
+		journeyFiles.answers[this.fieldName].uploadedFiles = [];
 
 		if (journeyResponse?.answers[this.fieldName]?.uploadedFiles) {
-			responseToSave.answers[this.fieldName].uploadedFiles.push(
+			journeyFiles.answers[this.fieldName].uploadedFiles.push(
 				...journeyResponse.answers[this.fieldName].uploadedFiles
 			);
 		}
 
 		if (uploadedFiles) {
-			responseToSave.answers[this.fieldName].uploadedFiles.push(...uploadedFiles);
+			journeyFiles.answers[this.fieldName].uploadedFiles.push(...uploadedFiles);
 		}
 
-		journeyResponse.answers[this.fieldName] = responseToSave.answers[this.fieldName];
+		journeyResponse.answers[this.fieldName] = journeyFiles.answers[this.fieldName];
+
+		uploadedFiles.forEach((file) => {
+			const data = {
+				...file,
+				type: this.documentType.name
+			};
+			apiClient.postSubmissionDocumentUpload(
+				journeyResponse.referenceId,
+				journeyResponse.answers.id,
+				data
+			);
+		});
```

Comment:
is `apiClient. postSubmissionDocumentUpload` async?

Reply:
yep, have added an await